tests/tests_fabric/strategies/test_xla_fsdp.py

```diff
@@ -245,11 +245,7 @@ def test_xla_fsdp_train_save_load(tmp_path, state_dict_type, use_auto_wrap_polic
     strategy = XLAFSDPStrategy(
         auto_wrap_policy=always_wrap_policy if use_auto_wrap_policy else None, state_dict_type=state_dict_type
     )
-    fabric = Fabric(
```

Comment:
why do you drop `devices` here?

Reply:
Fabric will take care of this automatically, so there is no need to specify device count to 4